heron/downloaders/src/java/org/apache/heron/downloader/DownloadRunner.java

```diff
@@ -127,6 +129,8 @@ private static Options constructCliOptions() {
 
   // takes topology package URI and extracts it to a directory
   public static void main(String[] args) throws Exception {
+    SLF4JBridgeHandler.removeHandlersForRootLogger();
+    SLF4JBridgeHandler.install();
```

Comment:
Can we use the `Slf4jUtils.installSLF4JBridge` that you have created for better maintainability? We can then switch over the import for `org.apache.heron.api.utils.Slf4jUtils`.

https://github.com/apache/incubator-heron/blob/0ad1b238708f50dd5979946689c59cefb8484636/heron/api/src/java/org/apache/heron/api/utils/Slf4jUtils.java

Great work!